test/verify/check-networkmanager-wireguard

```diff
@@ -140,7 +158,6 @@ class TestWireGuard(packagelib.PackageCase, netlib.NetworkCase):
         m2.execute(f"ip addr add {m2_ip6}/64 dev wg0")
         m2.execute(f"wg set wg0 peer {m1_pubkey} allowed-ips {m1_ip4}/32,{m1_ip6}")
 
-        b.click(f"#networking-interfaces button:contains('{iface_name}')")
```

Comment:
This change seems unrelated, what does it do?

Reply:
It visits are the interface details page, clicking the interaface name:

![image](https://github.com/cockpit-project/cockpit/assets/108616679/30c37024-9034-40df-9359-5190c8d537cb)

But now it's visiting the page much earlier [here](https://github.com/cockpit-project/cockpit/pull/19395/files#diff-1e9b8cca06a692aac8083ddc21489f716c3ead2ba9cfd7b88de882faab5dbdd4R132). So, it'd be already be in that page, when it reaches this line.

Reply:
Makes sense, thanks!